CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Unreleased
 
+## JavaScript
+- Fixed a bug where if any injected dependencies to a JavaScript (e.g. `ux:Name`, `ux:Dependency`, `dep:`) have names that correspond to EcmaScript 5.1 reserved words, these dependencies will now be available with a dollar sign `$` prefix (e.g. `$undefined`) instead of causing a script compile error.
```

Comment:
In what way? Because we would inject more symbols from the data-context? If so, fusetools/Uno#1193 would also be a backwards incompatible change, no? In fact, adding *anything* would be a problem.

If that's really the case, maybe we should add an API to be able to resolve things like this manually rather than $-prefixing ourselves? Something like:

```JavaScript
var dep = require("FuseJS/Dependency");
var $class = dep("class");
```

...and only implicitly import non-keyword dependencies?